src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -513,4 +513,7 @@ Use the dropdown to view and switch to other projects this file may belong to.</
   <data name="SynchronizingWithClassView" xml:space="preserve">
     <value>Synchronizing with {0}...</value>
   </data>
+  <data name="Loading_immediate_window" xml:space="preserve">
+    <value>Loading immediate window</value>
```

Comment:
Trivial: This code runs to initialize intellisense in the "watch" window too--maybe pick a more generic phrase like "Loading IntelliSense" or a second phrase for the watch window?